src/checkpoint/checkpoint_txn.c

```diff
@@ -1602,7 +1602,7 @@ __wt_txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[], bool waiting)
 
     /* Trigger the checkpoint cleanup thread to remove the obsolete pages. */
     if (checkpoint_cleanup)
```

Comment:
Should this variable be renamed too? I think probably this and the comment should be updated even though they're in this other source file.

Reply:
Done.

Reply:
Was this one updated @ershov?
Furthermore, we may need to update the `api_data.py` as the name `checkpoint_cleanup` is no longer relevant. But this api_data.py should probably be in a different PR.

Reply:
There are many stats that indicate checkpoint cleanup. Even those also need to be updated. We also need a T2 ticket after those stats are modified.

Reply:
I think this work is much bigger than I anticipated.

Reply:
Yes, in the architecture document also, there are some places where we mention checkpoint cleanup; we need to change that also. Moving this code into a different folder from the btree folder would be good.